plugins/internal/tengoutil/secure_script.go

```diff
@@ -30,3 +36,66 @@ func NewSecureScript(input []byte, globals map[string]interface{}) (*tengo.Scrip
 
 	return s, nil
 }
+
+func createHTTPModule() map[string]tengo.Object {
+	return map[string]tengo.Object{
+		"get": &tengo.UserFunction{
+			Name: "get",
+			Value: func(args ...tengo.Object) (tengo.Object, error) {
+				if len(args) < 1 || len(args) > 2 {
+					return nil, fmt.Errorf("expected 1 or 2 arguments, got %d", len(args))
+				}
+
+				url, ok := tengo.ToString(args[0])
+				if !ok {
+					return nil, fmt.Errorf("expected argument 1 (URL) to be a string")
+				}
+
+				headers := make(map[string]string)
+				if len(args) == 2 {
+					headerMap, ok := args[1].(*tengo.Map)
+					if !ok {
+						return nil, fmt.Errorf("expected argument 2 (headers) to be a map")
```

Comment:
we can use `errors.New`, or if you want, you can create a variable for the length, and use in this line. this is to avoid magic number. example:

```go
if len(args) == expectedArgsLength {
...
```

and the error can be:

```go
fmt.Errorf("expected argument %d (headers) to be a map", expectedArgsLength)
```

feel free to come up with a better name